css/properties.json

```diff
@@ -4712,6 +4640,49 @@
     "order": "uniqueOrder",
     "status": "standard"
   },
+  "offset": {
+    "syntax": "&lt;'offset-path'&gt; &amp;&amp; &lt;'offset-distance'&gt; &amp;&amp; &lt;'offset-rotation'&gt; &amp;&amp; &lt;'offset-anchor'&gt;",
+    "media": "visual",
+    "inherited": false,
+    "animationType": "discrete",
+    "percentages": [
+      "offset-distance"
+    ],
+    "groups": [
+      "CSS Motion"
+    ],
+    "initial": [
+      "offset-path",
+      "offset-distance",
+      "offset-rotation",
+      "offset-anchor"
+    ],
+    "appliesto": "allElementsSVGContainerElements",
```

Comment:
[According to the spec.](https://drafts.fxtf.org/motion-1/#propdef-offset) it applies to transformable elements.

Sebastian